### ISSUE TITLE
Apostrophe in list title turns into code

### ISSUE THREAD

**Issue by stopregionblocking** [internetarchive/openlibrary#8358]
I made a list titled "ߒߞߏ (N'Ko literature)". However, on many (but not all?) pages, it displays as "`ߒߞߏ (N&#39;Ko literature)`".

### Evidence / Screenshot (if possible)

![ID](https://github.com/internetarchive/openlibrary/assets/97799591/7d940228-e49a-42ec-ab81-4be4af2c168f)

### Relevant url?

https://openlibrary.org/people/holdee/lists/OL235009L/%DF%92%DF%9E%DF%8F_%28N_39_Ko_literature%29

### Steps to Reproduce
1. Make a list that has an apostrophe in the title
2. Go to the list page or view the page of an author on the list

### Details

- **Logged in (Y/N)?**
- **Browser type/version?**
- **Operating system?**
- **Environment (prod/dev/local)?** prod

**Comment by thinkGrow** [comment]
Hello. I cannot access the relevant URL. Anyway, I cannot completely assure you that I can solve it, but I can give it a shot.

**Comment by TanyaKansal** [comment]
the above issue works fine for me. I don't get the above issue even if I include ' in list title. plz check screenshot below:
![image](https://github.com/internetarchive/openlibrary/assets/41982971/6adf8c63-560b-4fc6-ab9b-b20fd586a8e3)

**Comment by davidscotson** [comment]
It might be related to the unicode text before the apostrophe.

I tried creating two lists, one with the exact same title as above, and one with just `ߒߞߏ` and while both display okay, I get an internal server error when trying to edit either list.

**Comment by digipakman** [comment]
> It might be related to the unicode text before the apostrophe.
> 
> I tried creating two lists, one with the exact same title as above, and one with just `ߒߞߏ` and while both display okay, I get an internal server error when trying to edit either list.

Unicode seems likely. Errors encoding the header for latin-1 are present when running locally related to the 500, haven't figured out what is different than prod yet but between store/retrieval it is handing the encoding/decoding a bit different

```
 0.0 (1): SELECT * FROM store WHERE key='account/openlibrary'
openlibrary-web-1           | [2023-10-09 04:23:23 +0000] [11] [ERROR] Error handling request /people/openlibrary/lists/OL4L/%E2%88%9Edfgd?m=edit
openlibrary-web-1           | Traceback (most recent call last):
openlibrary-web-1           |   File "/home/openlibrary/.local/lib/python3.11/site-packages/gunicorn/workers/sync.py", line 136, in handle
openlibrary-web-1           |     self.handle_request(listener, req, client, addr)
openlibrary-web-1           |   File "/home/openlibrary/.local/lib/python3.11/site-packages/gunicorn/workers/sync.py", line 185, in handle_request
openlibrary-web-1           |     resp.write(item)
openlibrary-web-1           |   File "/home/openlibrary/.local/lib/python3.11/site-packages/gunicorn/http/wsgi.py", line 326, in write
openlibrary-web-1           |     self.send_headers()
openlibrary-web-1           |   File "/home/openlibrary/.local/lib/python3.11/site-packages/gunicorn/http/wsgi.py", line 322, in send_headers
openlibrary-web-1           |     util.write(self.sock, util.to_bytestring(header_str, "latin-1"))
openlibrary-web-1           |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
openlibrary-web-1           |   File "/home/openlibrary/.local/lib/python3.11/site-packages/gunicorn/util.py", line 565, in to_bytestring
openlibrary-web-1           |     return value.encode(encoding)
openlibrary-web-1           |            ^^^^^^^^^^^^^^^^^^^^^^
openlibrary-web-1           | UnicodeEncodeError: 'latin-1' codec can't encode character '\u221e' in position 213: ordinal not in range(256)
```

**Comment by benbdeitch** [comment]
I gave this a brief look on my end, and was able to reproduce the problem successfulyl. I was able to edit the list title normally, and manually remove the &#39; without a problem.

<img width="477" alt="image" src="https://github.com/internetarchive/openlibrary/assets/131627264/2106e6f7-1aab-43b4-b1cb-fd0aba503caa">
I think we might have two issues going on here, if there was an inability to edit the list. Still, I'd be happy to work on the described problem in the title. Mind assigning it to me?

**Comment by benbdeitch** [comment]
I gave this a brief look on my end, and was able to reproduce the problem successfulyl. I was able to edit the list title normally, and manually remove the &#39; without a problem.

<img width="477" alt="image" src="https://github.com/internetarchive/openlibrary/assets/131627264/2106e6f7-1aab-43b4-b1cb-fd0aba503caa"><br>
I think we might have two issues going on here, if there was an inability to edit the list. Still, I'd be happy to work on the described problem in the title. Mind assigning it to me, @jimchamp ?
